packages/terra-application-navigation/docs/about.md

```diff
@@ -0,0 +1,20 @@
+# Terra Application Navigation
+
+The Terra Application Navigation is a responsive, themeable layout for building applications.
+
+The Terra Application Navigation provides:
+- A themeable, responsive application header, capable of rendering application-specific branding, tabular navigation, and user-centric utility controls
+- Responsive menu and content areas
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application-navigation`
```

Comment:
We should add peer dep info here

Reply:
I don’t think terra-application-navigation would be installed as a peer dependency

Reply:
I think i get what you're saying now

```suggestion
  - `npm install terra-application-navigation`
 
 - Install peer dependencies required by terra-application-navigation
  - `npm install react`
  - `npm install react-dom`
  - `npm install react-intl`
  - `npm install terra-application`
```

Reply:
If you rename this file to README.md and add the following comment, the peer dependency info for this package will be auto-generated when the docs:peer-dependencies script is run:

```
<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
<!-- AUTO-GENERATED-CONTENT:END -->
```